mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -167,14 +167,37 @@ object PCAModel extends MLReadable[PCAModel] {
 
     private val className = classOf[PCAModel].getName
 
+    /**
+     * Loads a [[PCAModel]] from data the input path. Note that the model includes an
```

Comment:
isnt there a word missing like: "Loads a PCAModel from data **located at** the input path"?